Enterprise-CMCS/macpro-onemac
# OY2 23578: Update CMS Package Details pages with CPOC + SRT fields from SEA Tool

Story: https://qmacbis.atlassian.net/browse/OY2-23578
Story: https://qmacbis.atlassian.net/browse/OY2-23647
Endpoint: See github-actions bot comment

### Details
CPOC (CMS Point Of Contact) is a field from SEA Tool that OneMAC would like to display on the details pages and as a column on the dashboard.  This is for all users, not just CMS (see [23647](https://qmacbis.atlassian.net/browse/OY2-23647))

SRT is the Subject Review Team - a list of contacts within CMS who are reviewing the package.  This list is only shown to CMS users and has a new format, a expandable list of names.

### Changes
- Updated the one-seed.json to include the CPOC and SRT in the package seed items.
- include the CPOC and the SRT team in the package item
- pass the CPOC in both user responses
- pass the SRT list in CMS user responses
- add the CPOC as a default package detail into the UI
- add the SRT as a new component utilizing the detail design

### Implementation Notes
Because the SRT is the final detail in the Detail Section, we can leave the current more-generic detail processor as-is and add the SRT as a state-aware component with the same design to the end of the details.  If we end up needing to alter this order... we will need to refactor how the detail Reviews are constructed and allow them to contain components with state.

### Test Plan
1. Login as a state submitting user
2. Navigate to dashboard
3. Verify that CPOC Name is a column that can be shown/hidden with default shown
4. Verify that clicking into the package shows the CPOC as a detail per the Figma and story AC
5. Verify that the Review Team (SRT) is NOT shown to the state user.
6. Logout/Login as a CMS user
7. Navigate to dashboard
8. Verify that CPOC Name is a column that can be shown/hidden with default shown
9. Verify that clicking into the package shows the CPOC as a detail per the Figma and story AC
10. Verify that the Review Team (SRT) IS shown to the CMS user.
11. Verify that if the Review Team contains more than 3 names, it is shrunk with an option to expand/show all
12. Verify that show/hide the Review Team works as expected.
